devicon.css

```diff
@@ -1,1134 +1,2048 @@
 @font-face {
-  font-family: 'devicon';
-  src:  url('fonts/devicon.eot?f7gwp9');
-  src:  url('fonts/devicon.eot?f7gwp9#iefix') format('embedded-opentype'),
-    url('fonts/devicon.ttf?f7gwp9') format('truetype'),
-    url('fonts/devicon.woff?f7gwp9') format('woff'),
-    url('fonts/devicon.svg?f7gwp9#devicon') format('svg');
+  font-family: "devicon";
+  src: url("fonts/devicon.eot?f7gwp9");
+  src: url("fonts/devicon.eot?f7gwp9#iefix") format("embedded-opentype"),
+    url("fonts/devicon.ttf?f7gwp9") format("truetype"),
+    url("fonts/devicon.woff?f7gwp9") format("woff"),
+    url("fonts/devicon.svg?f7gwp9#devicon") format("svg");
   font-weight: normal;
   font-style: normal;
   font-display: block;
 }
-
-[class^="devicon-"], [class*=" devicon-"] {
-  /* use !important to prevent issues with browser extensions that change fonts */
-  font-family: 'devicon' !important;
+[class^="devicon-"],
+[class*=" devicon-"] {
+  font-family: "devicon" !important;
   speak: never;
   font-style: normal;
   font-weight: normal;
   font-variant: normal;
   text-transform: none;
   line-height: 1;
-
-  /* Better Font Rendering =========== */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
-
 .devicon-nixos-plain-wordmark:before {
-  content: "\e992";
+  content: "";
 }
 .devicon-nixos-plain:before {
-  content: "\e9b2";
+  content: "";
 }
 .devicon-perl-plain:before {
-  content: "\e9b3";
+  content: "";
 }
 .devicon-tensorflow-line-wordmark:before {
-  content: "\e9b4";
+  content: "";
 }
 .devicon-tensorflow-line:before {
-  content: "\e9b5";
+  content: "";
 }
 .devicon-tensorflow-original-wordmark:before {
-  content: "\e9b6";
+  content: "";
 }
 .devicon-tensorflow-original:before {
-  content: "\e9b7";
+  content: "";
 }
 .devicon-zig-plain-wordmark:before {
-  content: "\e9b8";
+  content: "";
 }
-.devicon-zig-original:before {
-  content: "\e9b9";
+.devicon-zig-original:before,
+.devicon-zig-plain:before {
+  content: "";
 }
 .devicon-raspberrypi-line-wordmark:before {
-  content: "\e9ba";
+  content: "";
 }
 .devicon-raspberrypi-line:before {
-  content: "\e9bb";
+  content: "";
```

Comment:
For me, this just displays a weird emoji/text.
I have no idea what that could be. 🤷‍♂️

![image](https://user-images.githubusercontent.com/51391473/127046530-25fa365a-4e1c-44eb-bfaf-4e31806dcf55.png)

Reply:
Same for me, I cant figure out how to fix it, maybe it's just vscode acting up?

Reply:
Please consider fixing that problem. Thanks.

Reply:
Yeah I found it, I have to manually fix each one so that will be fun